proposals/automating-releases.md

```diff
@@ -0,0 +1,185 @@
+---
+title: automating-releases
+authors:
+  - @estroz
+reviewers:
+  - @jmrodri
+  - @joelanford
+  - @theishah
+approvers:
+  - TBD
+creation-date: 2020-09-19
+last-updated: 2020-09-24
+status: implementable
+---
+
+# automating-releases
+
+## Release Signoff Checklist
+
+- \[ \] Enhancement is `implementable`
+- \[ \] Design details are appropriately documented from clear requirements
+- \[ \] Test plan is defined
+- \[ \] Graduation criteria for dev preview, tech preview, GA
+- \[ \] User-facing documentation is created [for the website][operator-sdk-doc]
+
+## Open Questions
+
+1. We currently build images for an architecture on machines of that architecture.
```

Comment:
I think the go portion builds for various arches, the python stuff just gets installed for the various arches of the image.

Reply:
I'm looking into how we can use the [`docker buildx`](https://github.com/docker/buildx#building-multi-platform-images) plugin to build multi-arch images on a single machine of any arch.